### PR TITLE
Feature/team member page

### PR DESCRIPTION
**Pull request by NiFroPP**

### CODE REVIEW

File: src/components/common/PlayerCard/PlayerCard.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  return (
+		<div className={styles.card}>
+      <div className={styles.content}>
+        <div className={styles.member}>        
+          <div className={styles.avatar}>
+            <img src={ avatar || unknowAvatar} alt="avatar" />
+          </div>
+          <div className={styles.player}>
+            { 
+              role === RolesUsersEnum.ADMIN && <div className={styles.admin}>Admin</div>
```
Comment (Polina04) [inline src/components/common/PlayerCard/PlayerCard.tsx]: please create a constant before return
`const isAdmin = role === RolesUsersEnum.ADMIN;`
and use it here:
`{isAdmin && <div/>....}`

File: src/components/common/PlayerCard/PlayerCard.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            }
+            <div className={styles.name}>{`${firstName} ${lastName}`}</div>
+            <div className={styles.position}>{job}</div>
+        </div>
+        </div>
+        <div className={styles.delete}>
+          { role === RolesUsersEnum.REGULAR && <div>
```
Comment (Polina04) [inline src/components/common/PlayerCard/PlayerCard.tsx]: `const isRegularUser = role === RolesUsersEnum.REGULAR;`
the same as isAdmin

File: src/pages/TeamMembers/index.tsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+import { useTypedSelector } from "hooks/useTypedSelector";
+
+import styles from './index.module.scss';
+
+const TeamMembers: React.FC = () => {
+  const { members } = useTypedSelector<IUsersState>(state => state.membersReducer);
+
+  const admin = members.find((member) => member.role === RolesUsersEnum.ADMIN);
+  const users = members.filter((member) => member.role === RolesUsersEnum.REGULAR);
```
Comment (Polina04) [inline src/pages/TeamMembers/index.tsx]: `admin` and `users` are good places to use useMemo() hook

File: src/pages/TeamMembers/index.tsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+import MyButton from "components/common/MyButton/MyButton";
+
+import { IUsersState, RolesUsersEnum } from "redux/reducer/userReducer/types";
+
+import { useTypedSelector } from "hooks/useTypedSelector";
+
+import styles from './index.module.scss';
+
+const TeamMembers: React.FC = () => {
+  const { members } = useTypedSelector<IUsersState>(state => state.membersReducer);
```
Comment (Polina04) [inline src/pages/TeamMembers/index.tsx]: please create a file selectors.ts in redux directory and create all selector function there
`export const getMembers = state => state.membersReducer;`
and use them in all your app like:
`const { members } = useTypedSelector<IUsersState>(getMembers);`

**Review by DmitriyHnatiuk — APPROVED** [review]

File: src/components/common/PlayerCard/PlayerCard.tsx (hunk 1 above)
Comment (NiFroPP) [reply src/components/common/PlayerCard/PlayerCard.tsx]: fixed

File: src/components/common/PlayerCard/PlayerCard.tsx (hunk 2 above)
Comment (NiFroPP) [reply src/components/common/PlayerCard/PlayerCard.tsx]: fixed

File: src/pages/TeamMembers/index.tsx (hunk 3 above)
Comment (NiFroPP) [reply src/pages/TeamMembers/index.tsx]: fixed

File: src/pages/TeamMembers/index.tsx (hunk 4 above)
Comment (NiFroPP) [reply src/pages/TeamMembers/index.tsx]: fixed

**Comment by DmitriyHnatiuk** [comment]
Great job

**Review by SyleimanovSergey — APPROVED** [review]

**Pull request merged**